From 4954a3a0966dc07ffe465c197799d22a5072e52e Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Tue, 3 Jan 2023 15:44:10 +0000
Subject: [PATCH 0402/1085] media: dt-bindings: Add DW9817 to DW9807 binding

The DW9817 is programmatically the same as DW9807, but
the output drive is a bi-directional -100 to +100mA
instead of 0-100mA.

Add the appropriate compativle string.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 .../bindings/media/i2c/dongwoon,dw9807-vcm.yaml      | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

--- a/Documentation/devicetree/bindings/media/i2c/dongwoon,dw9807-vcm.yaml
+++ b/Documentation/devicetree/bindings/media/i2c/dongwoon,dw9807-vcm.yaml
@@ -5,18 +5,24 @@
 $id: http://devicetree.org/schemas/media/i2c/dongwoon,dw9807-vcm.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: Dongwoon Anatech DW9807 voice coil lens driver
+title: Dongwoon Anatech DW9807 and DW9817 voice coil lens driver
 
 maintainers:
   - Sakari Ailus <sakari.ailus@linux.intel.com>
 
 description: |
   DW9807 is a 10-bit DAC with current sink capability. It is intended for
-  controlling voice coil lenses.
+  controlling voice coil lenses. The output drive is 0-100mA.
+  DW9817 is very similar as a 10-bit DAC with current sink capability,
+  however the output drive is a bidirection -100 to +100mA.
+
 
 properties:
   compatible:
-    const: dongwoon,dw9807-vcm
+    items:
+      - enum:
+          - dongwoon,dw9807-vcm
+          - dongwoon,dw9817-vcm
 
   reg:
     maxItems: 1
